.swift-version

```diff
@@ -1 +1 @@
-3.0
+4.0.3
```

Comment:
This should always just show major versions. Should just be `4.0`

Reply:
Nice catch!

Reply:
I am going to change this ASAP.